docs/cardinal/game/configuration.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: 'Configuration'
+description: 'Configuring the Cardinal game shard'
+---
+
+Each service in the World Engine stack reads its configuration values from environment variables. Using the [World CLI](https://github.com/Argus-Labs/world-cli) makes configuring your project even easier via its `world.toml` file, where all World Engine configuration values are in one place.
+
+Below are the key value pairs that can be set in either the `world.toml` file or directly in environment variables.
+
+
+## Cardinal `[cardinal]`
+| Name                         | Description                                                                                                                                                                                                          | Required for Production |
+|------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------|
+| CARDINAL_NAMESPACE           | The name of your cardinal game shard. If you are running multiple game shards in a single World Engine instance, this MUST be unique for each game shard. Must only be alphanumeric characters.                      | YES                     |
```

Comment:
Nit: The namespace may also contain dashes.

Reply:
fixed [nits](https://github.com/Argus-Labs/world-engine/pull/736/commits/4e49fffad477551015c40df5da722fea99730694)